doc/examples/examples.hpp.in

```diff
@@ -225,6 +278,14 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *   </tr>
  *
  *   <tr valign="top">
+ *     <td> Iterative refinement
+ *     </td>
+ *     <td>@ref iterative_refinement,
+ *         @ref ir_ilu_preconditioned_solver
```

Comment:
This example can also be added to `Using preconditioners` a few lines above.
For the same category, we can add:
`adaptiveprecision_blockjacobi`
`par_ilu_convergence`
`preconditioner_export`.

Reply:
Both, ie ``iterative_refinemen`` and ``ir_ilu_preconditioned_solver`` to ``Using preconditioners``?

Reply:
no for iterative_refinement but yes for ir_ilu_preconditioned_solver, I think

Reply:
``ir_ilu_preconditioned_system`` is already added. I was not sure about ``iterative_refinement``. Fine. Thanks!